test/jdk/javax/net/ssl/templates/TLSBase.java

```diff
@@ -239,7 +243,7 @@ static class Server extends TLSBase {
         // test or the test will never end.
         void done() {
             try {
-                t.interrupt();
+                t.join(5000);
```

Comment:
I had to increase the join value of a test recently that failed with virtual threads.
Try adding `--test-make-args JTREG_TEST_THREAD_FACTORY=Virtual` to your mach5 command line.

Reply:
I'll check it out.. thanks

Reply:
Test passed with virtual threads on linux, windows, & mac aarch64